ldm/invoke/generator/inpaint.py

```diff
@@ -96,30 +104,46 @@ def make_image(x_T):
                 mask                       = mask_image,
                 init_latent                = self.init_latent
             )
+            return self.sample_to_image(samples)
+
+        return make_image
 
-            # Get PIL result
-            gen_result = self.sample_to_image(samples).convert('RGB')
+    def sample_to_image(self, samples)->Image:
+        gen_result = super().sample_to_image(samples).convert('RGB')
+        
+        pil_mask = self.pil_mask
+        pil_image = self.pil_image
+        mask_blur_radius = self.mask_blur_radius
 
-            # Get numpy version
-            np_gen_result = np.asarray(gen_result, dtype=np.uint8)
+        # Get the original alpha channel of the mask
+        pil_init_mask = pil_mask.convert('L')
```

Comment:
The UI is currently passing RGBA as the mask. Could call it a bug I guess, and make the UI fix it.